### PR TITLE
Fix: emotion issue

### PR DESCRIPTION
**Pull request by nelsW70**
- pinned Node version via Volta
- updated eslintrc
- updated tsconfig
- updated imports of emotion

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -66,6 +66,7 @@
       }
     ],
     "react/display-name": 0,
+    "react/no-unknown-property": ["error", { "ignore": ["css"] }],
```
Comment (tborres) [inline .eslintrc:69]: ❓ Was this per emotion's documentation?

File: tsconfig.json [hunk 2]
```diff
@@ -17,8 +18,8 @@
     "skipLibCheck": true,
     "strict": true,
     "target": "es5",
-    "incremental": true,
-    "types": ["@emotion/core"]
+    "types": ["@emotion/react/types/css-prop"],
+    "incremental": true
```
Comment (tborres) [inline tsconfig.json:22]: ❓ Does this need to come at the end, or can these be placed in alpha-order?

**Review by tborres — APPROVED** [review]
Everything seems to be running correctly. Just a couple of sanity-check questions. Thanks for handling this.

File: .eslintrc (hunk 1 above)
Comment (nelsW70) [reply .eslintrc:69]: @tborres, was looking at that before you commented. I don't think it's necessary with the update to the tsconfig. 

Just removed it and am not seeing any issues

File: tsconfig.json (hunk 2 above)
Comment (nelsW70) [reply tsconfig.json:22]: @tborres, there don't seem to be any issues alpha-ordering them

**Pull request merged**